Replace `user.group` chown syntax with `user:group`

The former is no longer supported by busybox chown as of 1.37.0.
GNU coreutils also issues a warning on its use.

--- a/Makefile
+++ b/Makefile
@@ -41,11 +41,11 @@
 install:
 	mkdir -p $(DATA_PREFIX)
 	cp -r data/* $(DATA_PREFIX)
-	chown -R root.root $(DATA_PREFIX)
+	chown -R root:root $(DATA_PREFIX)
 	chmod -R a+rX,g-w,o-w $(DATA_PREFIX)
 	mkdir -p $(BIN_PREFIX)
 	cp bugsquish $(BIN_PREFIX)
-	chown root.root $(BIN_PREFIX)/bugsquish
+	chown root:root $(BIN_PREFIX)/bugsquish
 	chmod a+rx,g-w,o-w $(BIN_PREFIX)/bugsquish
 
 
